### ISSUE TITLE
Big `buffer` dependency

### ISSUE THREAD

**Issue by moritzjacobs** [datocms/react-datocms#45]
The image component uses [`buffer`](https://www.npmjs.com/package/buffer)s global to isomorph `btoa()` ([here](https://github.com/datocms/react-datocms/blob/84503140f018a8c94d7ebb48cfe86f8e51ddedd2/src/Image/index.tsx#L5-L7)) which creates a rather large dependency in the client bundle.

If I'm not mistaken you could switch that out with something more lightweight, like [universal-base64](https://www.npmjs.com/package/universal-base64) to improve client load.

```

 MODULE  buffer
├─ imported: 1 time
├─ deps count: 2
├─ size: 49 KiB [for all included files]
├─ type: [transitive]
│  └─ buffer -> react-datocms
│  
├─ chunks: 580
├─ locations: 
│  └─ ./node_modules/buffer/
│  
├─ files: 
│  └─ ./node_modules/buffer/index.js
│  
└─ reasons:
   └─ react-datocms
      └─ ./node_modules/react-datocms/dist/esm/Image/index.js + 1 modules  16:30-36  [provided]
```

Thanks for your consideration!

**Comment by stefanoverna** [comment]
Fixed in 3.0.2, thanks @moritzjacobs for pointing that out!

**Comment by metart43** [comment]
@stefanoverna swtiching to [universal-base64](https://www.npmjs.com/package/universal-base64) library started causing issues on Windows based OS. Specififcaly for IE and Firefox browsers.

I get `Invalid character` error in `node_modules/universal-base64/dist/browser.js:11:12`
